packages/cli/src/commands/generate/util/tailwind/tailwind.js

```diff
@@ -87,6 +87,15 @@ export const handler = async ({ force }) => {
 
         if (!configExists || force) {
           await execa('yarn', ['tailwindcss', 'init'])
+
+          // opt-in to upcoming changes
```

Comment:
I agree and recommend we opt-in to upcoming changes. And, given that we use a different directory structure than tailwind init default, we should keep the setup steps as is (and not use the `tailwindcss init -p` flag.

However, I just merged this https://github.com/redwoodjs/redwood/pull/1234 which moves this code from `util/` to `setup/`. So please move it into the new file and we'll get this merged 🚀